src/Templates/AllWidget/AllWidget.js

```diff
@@ -35,17 +35,17 @@ export class AllWidget {
   }
 
   start(path) {
-    return new Promise( resolve => {
+    return new Promise((resolve) => {
```

Comment:
It seems this file only has coding style changes. Are those changes due to fact that prior to this version
the coding style was not applied to this file ?

Reply:
yes this change is due to prettier formatter plugin.

Reply:
I meant all the changes of this file seem to be coding style changes. That goes into the right direction.
Although what we want to avoid is going round in circles because different contributors have different coding styles (drawn from their respective editor). Hence my question: is _the_ coding of this code (as specified by the eslint configuration) now hardwired for this file ? For other files ? (this doesn't seem to be the case according to 
[the current version of .eslintrc.js](https://github.com/VCityTeam/UD-Viz/blob/master/.eslintrc.js))

In other terms shouldn't eslint be run in the CI to avoid such "noisy" side effects ?

Reply:
eslint is okay with the both version (with parenthesis and without) meaning that running the eslint CI will not throw error but prettier put parenthesis. Iam not sure @clementcolin who writed this code originally is using a formatter. If this is the case it's not a formatter conflict.

This highlight the fact that eslint and prettier are not completly synchronise see this issue https://github.com/VCityTeam/UD-Viz/issues/181

Reply:
I do use a formatter, hence it should be a "prettier" problem as eslint does not return any warning. Maybe there is a way to add a prettier to eslint ?